Cargo.toml

```diff
@@ -662,7 +662,7 @@ notify-types = { git = "https://github.com/zed-industries/notify.git", rev = "bb
 
 [profile.dev]
 split-debuginfo = "unpacked"
-debug = "limited"
+debug = "full"
```

Comment:
Nit: I guess we could just take this line out altogether, as it's the default for `dev` builds